src/main/groovy/com/bmuschko/gradle/java2html/ConvertCodeTask.groovy

```diff
@@ -30,23 +30,23 @@ import org.gradle.api.tasks.TaskAction
  */
 class ConvertCodeTask extends DefaultTask {
     @InputFiles FileCollection classpath
```

Comment:
Let's make this [`@Classpath`](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/Classpath.html).

Reply:
OK